ui/src/actions/citations.js

```diff
@@ -52,12 +53,17 @@ function fetchCitationsSummaryError(error) {
   };
 }
 
-export function fetchCitations(pidType, recordId, paginationOptions) {
+export function fetchCitations(pidType, recordId, newQuery = {}) {
```

Comment:
since you're at it, could you also remove this `pidType` since it's only `literature`

Reply:
oh ok, I thought it was like that because we foresee there will be something other than literature later